templates/www/cz/2019/devconfcz-2019-faq.html

```diff
@@ -186,6 +187,45 @@ <h2 id="speaker">{{ _('Speaker Questions') }}</h2>
+        <div class="toggle-content"><p>{{ _('The concept of the badge flair contest started at <a href="http://www.writethedocs.org/conf/prague/2018/badge-flair/">Write the Docs</a>, a conference and community for documentation-oriented tech folks who love to start fun and light-hearted traditions. The idea is simple: Decorate your badge, tweet a picture, and you might win a prize!') }}</p>
+	</div>
+	</div>
+
+	<div class="toggle">
+        <a href="#" class="toggle-title"><strong>{{ _('Q: Who can participate? Do I have to participate?') }}</strong><i class="fa fa-plus"></i></a>
+        <div class="toggle-content"><p>{{ _('All conference attendees and volunteers are eligible to participate. Organizing team members cannot enter the contest, although we do encourage you to lead by example and spread the word.') }}</p>
+          <p>{{ _('<u>The contest is optional.</u> Of course, you are free to use the decorating station even if you don’t want to participate, and decorate your badge for your own enjoyment!') }}</p>
+	</div>
+	</div>
+
+	<div class="toggle">
+        <a href="#" class="toggle-title"><strong>{{ _('Q: Sounds great! How do I participate?') }}</strong><i class="fa fa-plus"></i></a>
+        <div class="toggle-content">
+          <ul>
+            <li>{{ _('During the conference, we will have a decorating station near the registration desk with all kinds of stickers. Choose your favorite stickers and decorate your badge however you want.') }}</li>
+            <li>{{ _('Tweet a picture of your badge before <b>Saturday at 12:00</b>, using <i>both hashtags <b>#devconfcz</b> and <b>#devconfbadge</b></i>. If you do not have a Twitter account, you may ask a friend to tweet the picture for you; the prize will be given to the owner of the badge itself.') }}</li>
+            <li>{{ _('The jury will choose a winner, and the MC will announce it at <b>17:30 on Saturday, just before the lightning talks</b>. There might be an honorary mention as well.') }}</li>
+            <li>{{ _('<u>The winner must be in the room at the time of the announcement.</u> If they aren’t present, the prize will go to the runner-up. Follow the <a href="https://twitter.com/devconf_cz">DevConf CZ</a> Twitter account to stay updated with room announcements and other contest updates.') }}</li>
+          </ul>
+	</div>
+	</div>
+
+	<div class="toggle">
+        <a href="#" class="toggle-title"><strong>{{ _('Q: What are the decoration guidelines? Are there any restrictions?') }}</strong><i class="fa fa-plus"></i></a>
+        <div class="toggle-content"><p>{{ _('We want you to be creative and have fun, but we also want to make sure the contest is fair and inclusive. Please keep in mind the following guidelines:') }}</p>
+        <ul>
+          <li>{{ _('You should not cover any part of your name, but you can cover the DevConf logo or other elements if you don't need to acces them.') }}</li>
```

Comment:
It's prohibited to use  '  in the text inside of {{ _(' ') }}, replace it by ’ or use non-shorted version of the words (e.g. don't -> do not)

Reply:
for proper ’ see line 202 and word "don’t"